src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -324,6 +325,54 @@ protected virtual JObject SendToAddress(JArray _params)
             return SignAndRelay(tx);
         }
 
+        [RpcMethod]
+        protected virtual JObject InvokeContractVerify(JArray _params)
+        {
+            CheckWallet();
+            UInt160 script_hash = UInt160.Parse(_params[0].AsString());
+            ContractParameter[] args = _params.Count >= 2 ? ((JArray)_params[1]).Select(p => ContractParameter.FromJson(p)).ToArray() : new ContractParameter[0];
+            Signers signers = _params.Count >= 3 ? SignersFromJson((JArray)_params[2]) : null;
+            return GetVerificationResult(script_hash, args, signers);
+        }
+
+        private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] args, Signers signers = null)
+        {
+            var snapshot = Blockchain.Singleton.GetSnapshot();
+            var contract = NativeContract.ContractManagement.GetContract(snapshot, scriptHash);
+            if (contract is null)
+            {
+                throw new RpcException(-100, "Unknown contract");
+            }
+            var methodName = "verify";
+
+            Transaction tx = signers == null ? null : new Transaction
+            {
+                Signers = signers.GetSigners(),
+                Attributes = Array.Empty<TransactionAttribute>()
+            };
+            ContractParametersContext context = new ContractParametersContext(tx);
+            wallet.Sign(context);
+            tx.Witnesses = context.Completed ? context.GetWitnesses() : null;
+
+            using ApplicationEngine engine = ApplicationEngine.Create(TriggerType.Verification, tx, snapshot.Clone());
+            engine.LoadScript(new ScriptBuilder().EmitDynamicCall(scriptHash, methodName, true, args).ToArray(), (ushort)args.Length, 1);
```

Comment:
Please delete `ReturnValue` in EmitDynamicCall
```suggestion
            engine.LoadScript(new ScriptBuilder().EmitDynamicCall(scriptHash, methodName, args).ToArray(), (ushort)args.Length, 1);
```